src/components/FilteringPanel/Filter/Filter.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import React, { SetStateAction, useState } from 'react';
+import { Filter as FilterType } from '../../../types/Filter';
+
+type FilterProps = {
+  setFilter: React.Dispatch<SetStateAction<FilterType>>;
+};
+
+export const Filter: React.FC<FilterProps> = ({ setFilter }) => {
+  const [selectedLink, setSelectedLink] = useState<FilterType>('all');
```

Comment:
same with this state, you don't need it, just pass the filter state into this component